faults/aws/ec2-stop-by-tag/fault.yaml

```diff
@@ -60,7 +60,7 @@ spec:
     imagePullPolicy: Always
     args:
       - -c
-      - ./experiments -name ec2-stop-by-tag
+      - ./experiments -name ec2-terminate-by-tag
```

Comment:
is this because image has entry point with ec2-terminate-by-tag? Is there a plan to change to stop here also.>

Reply:
Yes, in the future we can put a PR in litmus-go to rename it completely.